backend/src/utils/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+//TODO find a way to enforce regex format for email inside an interface.
```

Comment:
Should this comment be removed?

Reply:
I guess yes, i got the enforment through AJV, so it should be fine, as long as we validate everything.